crates/primitives-traits/Cargo.toml

```diff
@@ -65,3 +65,4 @@ arbitrary = [
     "dep:proptest-arbitrary-interop",
 ]
 serde-bincode-compat = ["serde_with", "alloy-consensus/serde-bincode-compat"]
+optimism = []
```

Comment:
I really dont want to introduce an op feature in this crate because this crate was only created so we have one crate that's not polluted with op feautres.